src/components/tables/TableHeadingRow.tsx

```diff
@@ -14,6 +14,7 @@ export interface ITableHeadingRowOwnProps extends React.ClassAttributes<TableHea
     className?: string;
     isMultiSelect?: boolean;
     selectionDisabled?: boolean;
+    isPartOfCollapsibleTable?: boolean;

```

Comment:
isPartOfCollapsibleTable --> tableHasCollapsibleRows ? 🤔

Reply:
Thanks, I was clueless as to how to name this :stuck_out_tongue: